packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -428,9 +429,8 @@ export async function createEmulator(displayName: string, systemImage: AndroidSy
     ["hw.cpu.arch", getNativeQemuArch()],
     ["hw.cpu.ncore", "4"],
     ["hw.dPad", "no"],
-    ["hw.device.hash2", "MD5:3db3250dab5d0d93b29353040181c7e9"],
     ["hw.device.manufacturer", "Google"],
-    ["hw.device.name", "pixel_7"],
```

Comment:
I managed to reproduce the issue with misaligned front camera, but it was flaky - sometimes it worked, sometimes not, but I believe this change fixed it as I'm not able to reproduce it now :P 

As for other misalignments, it looks like there is an issue with skin/mask in Android Studio itself, when I run Pixel 9 emulator from command line it also looks not that well:

<img width="396" alt="Zrzut ekranu 2024-10-10 o 16 58 09" src="https://github.com/user-attachments/assets/5f9eca32-11ca-45e2-83e7-094c0457aac8">

<img width="484" alt="Zrzut ekranu 2024-10-10 o 16 52 17" src="https://github.com/user-attachments/assets/6cdfd02f-98ee-430a-8d77-9f2b0892278c">

<img width="480" alt="Zrzut ekranu 2024-10-10 o 16 55 11" src="https://github.com/user-attachments/assets/bcbbe5d1-1bbd-4d60-8a31-f4a6ac8a7393">

<img width="463" alt="Zrzut ekranu 2024-10-10 o 16 57 49" src="https://github.com/user-attachments/assets/63f03fde-3ae5-48e3-834b-49e97c9b94b4">